### PR TITLE
Add `deploy-example-kubernetes` target to Makefile

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -233,10 +233,15 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 	make deploy-yaml | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
 .PHONY: deploy-example
-deploy-example: ## Deploy an example Istio resource
+deploy-example: ## Deploy an example Istio resource on OpenShift
```
Comment (jwendell) [inline Makefile]: perhaps we could rename this target to be `deploy-example-openshift` to keep the pattern?

File: Makefile [hunk 2]
```diff
@@ -233,10 +233,15 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
...
 	kubectl create ns istio-system || echo "namespace istio-system already exists"
 	kubectl apply -n istio-system -f config/samples/operator.istio.io_v1alpha1_istio.yaml
 
+.PHONY: deploy-example-kubernetes
+deploy-example-kubernetes: ## Deploy an example Istio resource on Kubernetes
+	kubectl create ns istio-system || echo "namespace istio-system already exists"
+	kubectl apply -n istio-system -f config/samples/operator.istio.io_v1alpha1_istio_kubernetes.yaml
```
Comment (jwendell) [inline Makefile]: Is this a good opportunity to simplify these example filenames?

File: Makefile (hunk 1 above)
Comment (luksa) [reply Makefile]: done

File: Makefile (hunk 2 above)
Comment (luksa) [reply Makefile]: done

**Review by jwendell — APPROVED** [review]

**Comment by jwendell** [comment]
@luksa the test failed with

```
 + oc apply -f /work/tests/integration/../../config/samples/operator.istio.io_v1alpha1_istio.yaml -n istio-system
error: the path "/work/tests/integration/../../config/samples/operator.istio.io_v1alpha1_istio.yaml" does not exist 
```

**Pull request merged**